docs/journalist.rst

```diff
@@ -248,33 +336,76 @@ default application.
 
 |Opened document|
 
-If the default application does not work, you can right-click on the
-document and choose **Open with Other Application...** to try opening
-the document with OpenOffice Writer, or Document Viewer. You might
-also need to right-click on a file and choose **Rename...** to rename
-a document with a proper file extension (for example, ".jpg" instead
-of ".jpeg").
-
 Working with Documents
 ----------------------
 
-This section describes how to handle unusual file formats, safely research
-submissions, remove metadata, and mitigate risks from submitted malware.
+This section describes how to organize submissions, handle unusual file formats,
```

Comment:
Although implied by this paragraph, I recommend  be slightly more explicit and state that opening random attachment will always carry  certain risks

Reply:
I've now expanded on the malware problem a bit in the malware section. The paragraphs starting with 
>"It is crucial, however,"
are new.